packages/lit-node-client-nodejs/src/lib/helpers/validate-bls-session-sig.ts

```diff
@@ -0,0 +1,28 @@
+import { AuthSig } from '@lit-protocol/types';
+import { uint8arrayToString } from '@lit-protocol/uint8arrays';
+import { ethers } from 'ethers';
+
+const LIT_SESSION_SIGNED_MESSAGE_PREFIX = 'lit_session:';
```

Comment:
are you sure it's `lit_session` but not `lit:session:`?

Reply:
Yes I confirmed this with @DashKash54 when i tried `lit-session:*` it failed.

Reply:
This `lit_session` is a hard coded value used when generating the BLS session signature.
see here for where it is used in [`lit-node`](https://github.com/LIT-Protocol/lit-assets/blob/add/abi-serialization-for-network-info/rust/lit-node/src/auth/auth_material.rs#L800)